### DISCUSSION TITLE
User Access Control plugin ? "Tony Musgrove"

### DISCUSSION THREAD

**Discussion by siliconhippy** [jellyfin/jellyfin#10202] [Q&A]
I have been looking at Users vs Admin, on my Jellyfin v 10.8.10.0 on my DietPi x86_64 server.

What I want is a "Managing User", below admin user, who cannot change or delete admin user, but who can manage, add or delete other 'regular' accounts to quite some extent ...

Google Bard and Bing Skype Chat referred to some community plugin (per title above) which seems to be part of Jellyfin github (where user/pwd access is required, of course.)

So is there a way to download and install this plugin (in case it does the above job), or is there any way to create Managing Users ?

**Comment by crobibero** [comment]
This plugin does not exist.

**Comment by siliconhippy** [comment]
Ok I thought so 😄
But is there any other way, e.g , some "simple" LDAP plugin config, or database editor, etc?

**Comment by pypp** [comment]
There is this plugin that might be helpful to you:
https://github.com/jellyfin/jellyfin-plugin-ldapauth

**Comment by siliconhippy** [comment]
Yes thanks, seen this 😀
But gets a bit complicated.
